search-resources-api/src/test/java/no/unit/nva/search/SearchResourcesApiHandlerTest.java

```diff
@@ -0,0 +1,76 @@
+package no.unit.nva.search;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.search.exception.InputException;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.handlers.RequestInfo;
+import nva.commons.utils.Environment;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+
+public class SearchResourcesApiHandlerTest {
+
+
+
+    @Test
+    @DisplayName("testDefaultConstructor")
+    public void testDefaultConstructor() {
+        SearchResourcesApiHandler searchResourcesApiHandler = new SearchResourcesApiHandler();
+        assertNotNull(searchResourcesApiHandler);
+    }
+
+
+    @Test
+    @DisplayName("testConstructorWithParameters")
```

Comment:
Test isn't necessary anymore as it is tested in every test

Reply:
Simplified tests